api/agentpb/agent.proto

```diff
@@ -326,6 +326,17 @@ message StartActionRequest {
     // Contains files and their contents which can be used in DSN.
     TextFiles text_files = 2;
   }
+
+  // RestartMongoDBServiceParams describes an action request to restart components in a MongoDB node
+  message RestartMongoDBServiceParams {
+    // Service is a service that can be restarted from PMM server
+    enum Service {
+      SERVICE_NAME_INVALID = 0;
+      MONGOD = 1;
+      PBM_AGENT = 2;
+    }
+    Service service = 1;
```

Comment:
For the current use case, we can't restart pbm-agent and mongod at once because pbm-agent wants all mongod instances to be available otherwise, it will fail (so an array won't work)